src/pages/workspaces/workspace/Notebooks.js

```diff
@@ -182,15 +182,59 @@ class WorkspaceNotebooks extends Component {
     const { name: wsName, namespace } = this.props
 
     return div({ style: { display: listView ? undefined : 'flex', flexWrap: 'wrap' } },
-      _.map(({ name, updated }) => h(NotebookCard, {
-        name, updated, listView, bucketName, namespace, wsName,
-        reloadList: () => this.refresh()
-      }), notebooks)
+      [
+        div({
+          style: {
+            margin: '1.25rem',
+            display: 'flex', flexDirection: listView ? 'row' : 'column',
+            width: listView ? undefined : 200, height: listView ? undefined : 250,
+            fontSize: listView ? 16 : undefined, lineHeight: listView ? '22px' : undefined
+          }
+        }, [
+          h(Interactive, {
+            as: 'div',
+            style: _.merge(Style.elements.card, { flexGrow: 1, color: Style.colors.secondary }),
```

Comment:
If you're trying to get the cards to be equal width/height, you probably want `flex: 1`. Otherwise the basis will stay `auto`, and the sizing will be change based on the content.

Reply:
`flex: 1` made them weird sizes. Keeping as is.